### PR TITLE
adding details on how to citation the package and specific models

### LINKED ISSUE #165: B92_MWAvg reference missing
Needs to be added to the references section of the docs.

### LINKED ISSUE #206: Citation guidelines
We should add some guidelines in the GitHub readme, and in the readthedocs on how to cite the package and the papers related to the different curves in the package.
Also, some more information on when to use what curve would likely be appreciated by the users.

### PR DESCRIPTION
**Pull request by karllark**
Closes #206
Closes #165

### CODE REVIEW

File: docs/dust_extinction/references.rst [hunk 1]
```diff
@@ -2,6 +2,43 @@
...
++--------------+----------------------+
+| O94          | CCM89                |
++--------------+----------------------+
+| G03          | GC98, MCG99          |
++--------------+----------------------+
+| G23          | GCC09, F19, G21, D22 |
++--------------+----------------------+
+
+In addition, please cite the dust_extinction package 
+(BibTex format found on "Cite this repository" button on main repository page).
```
Comment (mdecleir) [inline docs/dust_extinction/references.rst:24]: I am not sure if I know where to find this button. Is this on Github or read the docs?

File: docs/dust_extinction/references.rst [hunk 2]
```diff
@@ -2,6 +2,43 @@
...
+
+Gordon, K. (2024). dust_extinction (Version v1.4.1) [Computer software]. https://doi.org/10.5281/zenodo.11235336
+
+Naming Convention
+=================
+
+The names of models use the paper where they were presented and are
+based on the author last name initials and year published.
+For papers with 3 or fewer authors, the initials of all the authors are used.
+For papers with 4 or more authors, only the initial of the 1st authors is used.
```
Comment (mdecleir) [inline docs/dust_extinction/references.rst:34]:
```suggestion
For papers with 4 or more authors, only the initial of the 1st author is used.
```

**Review by mdecleir — COMMENTED** [review]
Just 2 small comments.
Otherwise, it looks great.
Thanks for this update!

**Comment by mdecleir** [comment]
One additional comment/question. What is the purpose of the reference API: https://dust-extinction.readthedocs.io/en/stable/dust_extinction/reference_api.html?

It looks like some of that page is broken.

File: docs/dust_extinction/references.rst (hunk 1 above)
Comment (karllark) [reply docs/dust_extinction/references.rst:24]: Github.  Will clarify.

File: docs/dust_extinction/references.rst (hunk 2 above)
Comment (karllark) [reply docs/dust_extinction/references.rst]: Fixed.

**Comment by karllark** [comment]
> One additional comment/question. What is the purpose of the reference API: https://dust-extinction.readthedocs.io/en/stable/dust_extinction/reference_api.html?
> 
> It looks like some of that page is broken.

This page provides the programming interface info.  There are supposed to be graphical flow diagrams.  Graphviz is clearly not working.  Will have to investigate.  Not part of this PR.  Will open an issue.

**Pull request merged**